index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+    <title>Portfolio</title>
+</head>
+<body class="body">
+
+<header class="toolbar">
+    <div class=left>
+        <p class=my-name>Mouaz El Molkey</p>
+         </div>
+        
+         <div class=right>
+            <a class=menu_item>Portoflio</a>
+            <a class=menu_item>About</a>
+            <a class=menu_item>Contact</a>
+            <img class="menu_mail" src="/img/mail.png" alt="Email">
+             </div>
+            
+</header>
+<section class="intro">
+<h1 class="big-name"> Mouaz El Molkey</h1>
+<img src="img/circle-1.png" class="circle-1" alt="">   
+<h2 class="small-name">Software Developer</h2>
+</section>
+
+<section class="main-section">
+    <h3 class="can-do">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me</h3>
```

Comment:
- [ ] This text should be wrapped inside a `p` tag and not an `h3` tag. Please fix that.